src/lang/en-US.ts

```diff
@@ -374,8 +374,12 @@ const CaptureCreate: ResolvedIntlConfig['messages'] = {
 const DetailsPanel: ResolvedIntlConfig['messages'] = {
     'detailsPanel.logs.title': `Logs`,
     'detailsPanel.logs.notFound': `Logs for this build cannot be found. This is likely a permissions issue. You don't have permissions to view other users' logs by default.`,
-    'detailsPanel.shardDetails.id.label': `Shard ID:`,
+    'detailsPanel.shardDetails.title': `Shard Information`,
+    'detailsPanel.shardDetails.status.label': `Status`,
+    'detailsPanel.shardDetails.id.label': `ID`,
```

Comment:
This is fine for now... but  I feel like this label might be a good one to put into the `commonmessages`